playbooks/roles/edx_django_service/tasks/main.yml

```diff
@@ -193,6 +193,17 @@
     - install
     - install:app-requirements
 
+- name: add extra requirements to extra.txt
```

Comment:
Is this something we should move to the ecommerce role instead, since this problem is unique to ecommerce?
Or do the meta tasks run before the ecommerce/tasks?

Reply:
@kaizoku, the `meta` tasks run before the `ecommerce` role's tasks, and this issue is common to many IDAs which use the `edx_django_service` role. So, it is useful to have it here so that we can enable it for other IDAs, if necessary.

Reply:
Thanks for clarifying @lgp171188. I glanced at a few other edX django apps but didn't see a similar setup with `tox` and thought it might be an ecommerce specific problem.